include/spirv-tools/libspirv.hpp

```diff
@@ -68,6 +68,8 @@ class Context {
 class ValidatorOptions {
  public:
   ValidatorOptions() : options_(spvValidatorOptionsCreate()) {}
+  ValidatorOptions(spv_validator_options options) : options_(options) {}
```

Comment:
explicit ValidatorOptions(...)

Reply:
Actually I do not need or want this one anymore, I'll undo this change, and add explicit to the default constructor.

Reply:
Default constructor doesn't need it, just single argument constructors.